.github/workflows/ci.yml

```diff
@@ -9,7 +9,13 @@
 #                   https://github.com/linux-nfc/neard
 #
 name: "Builds"
-on: [push, pull_request, workflow_dispatch]
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run at 1:01 AM, every Tuesday
```

Comment:
I know Dmitry hacks at night at lot, but moving this to daytime could accelerate developer report-fix-push turnout